rom/dev/tools/test-fmc/src/main.rs

```diff
@@ -268,6 +291,26 @@ fn read_pcr_log(mbox: &caliptra_registers::mbox::RegisterBlock<RealMmioMut>) {
     mbox.status().write(|w| w.status(|w| w.data_ready()));
 }
 
+fn read_measurement_log(mbox: &caliptra_registers::mbox::RegisterBlock<RealMmioMut>) {
+    let mut measurement_entry_count = 0;
+    loop {
+        let measurement_entry = get_pcr_entry(measurement_entry_count, true);
+        measurement_entry_count += 1;
+        send_to_mailbox(mbox, measurement_entry.as_bytes(), false);
+
+        if measurement_entry_count == MEASUREMENT_MAX_COUNT {
```

Comment:
The other log reading function tests whether `PcrLogEntryId::from(pcr_entry.id) == PcrLogEntryId::Invalid`. Any reason not to do the same here?

Reply:
The other log can contain up to 17 entries; currently only 11 PCRs are stored, so trying to read the 12th (invalid) entry does not result in an exception. This is not the case for the measurement log which can contain only 8 entries, hence the limit check.

Reply:
Ack. I think both routines should do both checks. A limit check to ensure we don't panic by reading past the end of the array, and an is-valid check to ensure we don't return more log entries than are actually populated (since as-is it looks like this will always return eight entries even if only a couple measurements were stashed).

Reply:
But will leave this as optional future work since this is just test code.